File: app/src/main/java/com/krystian/chessclock/CustomMatchDatabaseHelper.java
package com.krystian.chessclock;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

public class CustomMatchDatabaseHelper extends SQLiteOpenHelper {

    private final static String DB_NAME = "CUSTOM_MATCH_DATABASE";
    private final static int DB_VERSION = 1;

    public static final String MAIN_TABLE = "CUSTOM_MATCHES_TABLE"; //contains names of custom matches stored in other tables


    public CustomMatchDatabaseHelper(Context context) {
        super(context, DB_NAME, null, DB_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL("CREATE TABLE " + MAIN_TABLE +
                "(_id INTEGER PRIMARY KEY AUTOINCREMENT, " +
                "NAME TEXT, " +
                "NUMBER_OF_GAMES INTEGER);");
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
        if(DB_VERSION < 1) {
            onCreate(db);
        }
    }
}

File: app/src/main/java/com/krystian/chessclock/customMatchPackage/CustomMatchDatabaseHelper.kt
package com.krystian.chessclock.customMatchPackage

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class CustomMatchDatabaseHelper(context: Context?) :
    SQLiteOpenHelper(context, DB_NAME, null, DB_VERSION) {
    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL(
            "CREATE TABLE " + CustomMatchDatabase.CUSTOM_MATCHES_TABLE +
                    "(_id INTEGER PRIMARY KEY AUTOINCREMENT, " +
                    CustomMatchDatabase.NAME + " TEXT, " +
                    CustomMatchDatabase.NUMBER_OF_GAMES + " INTEGER);"
        )
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
        if (DB_VERSION < 1) {
            onCreate(db)
        }
    }

    companion object {
        private const val DB_NAME = "CUSTOM_MATCH_DATABASE"
        private const val DB_VERSION = 1
    }
}
